solutions/cf-702C.mdx

```diff
@@ -327,4 +327,52 @@ int main()
 
 </CPPSection>
 
+<JavaSection>
+	
+```java
+import java.io.*;
+import java.util.*;
+
+public class Main {
+	//CodeSnip{Kattio}
+	public static void main(String[] args) {
+		Kattio io = new Kattio();
+
+		int n = io.nextInt(), m = io.nextInt();
+		int[] cities = new int[n];
+		TreeSet<Integer> towers = new TreeSet<>();
+		for (int i = 0; i < n; i++)	
+		{
+			cities[i] = io.nextInt();
+		}
```

Comment:
(optional)
for control flow statements, put opening brace on same line as declaration